pages/about.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="overflow-y-auto w-full">
```

Comment:
Making the top nav sticky was an intentional decision. Let's discuss this on Slack?